.github/workflows/test-run-codebuild.yml

```diff
@@ -0,0 +1,35 @@
+name: Test run-codebuild ci script
+
+on:
+  pull_request:
+    paths:
+      - 'ci/scripts/run-codebuild/**'
+  push:
+    branches:
+      - main
+    paths:
+      - 'ci/scripts/run-codebuild/**'
+
+jobs:
+  test-script-run-codebuild:
+    runs-on: ubuntu-20.04
+    name: Test script run-codebuild
+    env:
+      working-directory: ci/scripts/run-codebuild
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579
+      - name: Setup
+        uses: actions/setup-node@1f8c6b94b26d0feae1e387ca63ccbdc44d27b561
+        with:
+          node-version: 12.22.7
```

Comment:
We can take advantage of the matrix testing here to do node 14 as well (given 12 is EOL very soon). Example here https://github.com/alphagov/pay-logging-lambdas/blob/main/.github/workflows/ci.yml#L7

Reply:
I could add 16 as well and see if it passes :) If so win! If not I can leave it out :)

Reply:
There we go :) Tested on 12, 14, and 16 :) Passing in all with no changes

Reply:
I've allowed 14 and 16 to float up within the major version, since this is to run on node-runner and currently that's on 12.22.7 it makes sense to lock to the specific target, but where we don't have a specific target yet we may as well float to stay up to date within the LTS release.